babycenter/static/css/modules/_languages.scss

```diff
@@ -11,7 +11,9 @@
     &:last-child:after {
       content: "";
     }
-
-    color: #999;
+    .active {
+      color: #999;
+    }
+    color: $orange;
```

Comment:
Also, move to the top of the file, before the `&:after` stuff and before, `.active`